spec/controllers/profile_controller_spec.rb

```diff
@@ -12,7 +12,8 @@
     it 'should create a new profile if one does not exist' do
       @user = FactoryGirl.create(:user)
       @user.profile.destroy
-      get :show, user: @user.id
+      @user.reload
+      get :show, user_id: @user
```

Comment:
(Just noting that this was a bit of a mix-up -- with both preferences and profile, we had our P-words confused. All is good!)